src/schema/imported/chrome_settings_overrides.json

```diff
@@ -25,7 +25,7 @@
                 "search_url": {
                   "type": "string",
                   "format": "url",
-                  "pattern": "^https://.*$",
+                  "pattern": "^(https://|http://(localhost|127\\.0\\.0\\.1|\\[::1\\])).*$",
```

Comment:
This pattern is incorrect, I filed a bug in Firefox @ https://bugzilla.mozilla.org/show_bug.cgi?id=1651665

To fix this, add `(:\d+)?(/|$)`